product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/other_monitoring/index.mdx

```diff
@@ -43,7 +43,7 @@ You could leverage some Python code to read the S3 bucket, then use the AWS APIs
 
 ### Watching for logs on Azure
 
-You could leverage [Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) to read the Azure Blob Storage object, then use the API to upload the data to CloudWatch.
+You could leverage [Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) to read the Azure Blob Storage object, then use the API to upload the data to your custom monitoring solution.
```

Comment:
```suggestion
You could leverage [Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/functions-overview) to read the Azure Blob Storage object, then use the API to upload the data to your monitoring solution.
```

Reply:
I don't think we need "custom" here. @jasonparton , do you want to take a quick look at this?

Reply:
agree...we can remove "custom" and keep the rest